catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/ImageInputTransformerTest.java

```diff
@@ -505,39 +506,39 @@ public void testPiaimc() throws IOException, NitfFormatException {
 
             try (InputStream inputStream = new FileInputStream(nitfFile)) {
                 Metacard metacard = metacardFactory.createMetacard("csexraTest");
-                NitfSegmentsFlow nitfSegmentsFlow = new NitfParserInputFlow().inputStream(inputStream)
+                NitfSegmentsFlow nitfSegmentsFlow = new NitfParserInputFlowImpl().inputStream(inputStream)
                         .headerOnly();
                 transformer.transform(nitfSegmentsFlow, metacard);
                 assertThat(metacard.getAttribute(Isr.CLOUD_COVER)
                         .getValue(), is(70));
             }
 
         } finally {
-            nitfFile.delete();
+            //nitfFile.delete();
```

Comment:
I removed the comment but kept the finally block. The call to delete() will clean up the temp file.